### ISSUE TITLE
Populate Glossary Wiki Page

### ISSUE THREAD

**Issue by epierotti3** [hackforla/internship#306]
### Dependencies
ANY ISSUE NUMBERS THAT ARE BLOCKERS OR OTHER REASONS WHY THIS WOULD LIVE IN THE ICEBOX

### Overview
We need to find terms to fill into the TWE Glossary Wiki Page so that this page is no longer empty.

### Action Items
- [x] Define places to look for glossary words
- [x] Create spreadsheet within Google Drive 
- [ ] Look in resources below for terms that might need to be defined and fill into spreadsheet
- [ ] Bring to team meeting for review

### Resources
- [Glossary](https://github.com/hackforla/internship/wiki/glossary)
- [Glossary Draft spreadsheet](https://docs.google.com/spreadsheets/d/10omnFb1L8lM4n4Rz610UzI1E51ZMRG6tuVnphrYnay0/edit#gid=0) (located in Internships > wiki)
#### Places to look for glossary words
- Wiki
- Research Plans
- Figma
- Find WIKI page for your role [Internship Wiki, Join the team page](https://github.com/hackforla/internship/wiki/Joining-the-Team)

**Comment by Nat-nn** [comment]
Great. Thank you. I will look into it.

**Comment by epierotti3** [comment]
Update:
1. Progress: Issue created by @epierotti3 - left role as missing because this could be worked by any new team member regardless of role. @Nat-nn is working on this as of now after we reviewed the issue and addressed the blocker of what kind of terms should be put into the glossary. Answer: any term that is unclear to a TWE team member!
3. Blockers: N/A (Natalia - you can add blockers if they arise to your next update).
4. Availability: N/A (Natalia - you can fill this in on your next update)
5. ETA: N/A (Natalia - you can fill this in on your next update)

**Comment by ExperimentsInHonesty** [comment]
I updated the spreadsheet to have the branding.

**Comment by bradyse** [comment]
@Nat-nn 
Please provide update
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (if necessary): "Add any pictures that will help illustrate what you are working on."

**Comment by Nat-nn** [comment]
1. Progress:  Added 25 more terms, discussed some terms with the team on a prior meeting.
2. Blockers:  Need to confirm how many terms we want to include, need to verify the scope. 
3. Availability: 1-2 hours
4. ETA:  2 weeks (by 09/22/22) 
5. Pictures (if necessary): N/A

**Comment by ExperimentsInHonesty** [comment]
Notes from meeting 2022-09-08
- add where term appeared
- add who benefits categorization

**Comment by Nat-nn** [comment]
My update 09/15/22 :
1. Progress:   Updated who benefits  and added links; Added several new terms. 
2. Blockers:   Who benefits - need some clarification; Where located: Are we trying to find only 1 location or want to add all locations where we have it coming up? 
3. Availability: 2-3 hours
4. ETA:  2 weeks (by 09/22/22) 
5. Pictures (if necessary): N/A

**Comment by Nat-nn** [comment]
Discussed during the meeting: Scope of the project, terms needed. 
Next Steps: To update sources/ links

**Comment by Nat-nn** [comment]
Progress: Updated Glossary (added additional terms, Updated links and added term reference pages) 
Blockers: Need Team Lead to review, to confirm Ok to add to WIKI
Availability: 1-3 hours per week
ETA: After next UX team meeting
Pictures or links: 
https://docs.google.com/spreadsheets/d/10omnFb1L8lM4n4Rz610UzI1E51ZMRG6tuVnphrYnay0/edit#gid=0

**Comment by ExperimentsInHonesty** [comment]
Notes from meeting
- We can link the where found, to the location it was found in
- official link, refers to either the domain holder or a wiki article, etc.
- line returns can be made in sheets by using alt-enter

**Comment by bradyse** [comment]
@Nat-nn 

Please provide update
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (if necessary): "Add any pictures that will help illustrate what you are working on."

**Comment by Nat-nn** [comment]
Progress: Added additional terms, ready to review. 
Blockers: Need to get Team lead approval for general links.
Availability: 1-3 hours per week
ETA: After next UX team meeting
Pictures or links:
https://docs.google.com/spreadsheets/d/10omnFb1L8lM4n4Rz610UzI1E51ZMRG6tuVnphrYnay0/edit#gid=0

**Comment by bradyse** [comment]
### Setup for a Peer Review
- [ ] Check if the Issue Overview clearly defines why this document needs to be created or updated. If it is not clear, add that information to the overview in 1-2 sentences.
- [ ] In the Resources, clearly identify the link to the document being created or updated that requires a peer review (use subheadings if needed).
- [ ] Copy and paste the Action Items for Peer Review into a comment of the issue.
- [ ] Replace the text in square brackets with the name of the document that requires a peer review (use the same name as it is called in the Resources).

### Action Items for Peer Review
- [ ] Read the issue overview to learn about why this document was created.
- [ ] Locate [DOCUMENT THAT NEEDS UPDATED] in the Resources that needs reviewed.

**Comment by bradyse** [comment]
@Nat-nn 

I have added the actions items for a peer review. Now we need to assign a peer reviewer to your issue. Perhaps @adarosh or @lkawentel would be interested in conducting this peer review (assuming that they don't currently have an issue to work on). If one of them cannot do it, then I would reach out to some of the other team members.

**Comment by Nat-nn** [comment]
Progress: Updated terms and links, moved project to 'peer review needed' status
Blockers: Wailing for "Peer Review"
Availability: 1-3 hours per week
ETA: After next UX team meeting
Pictures or links:
https://docs.google.com/spreadsheets/d/10omnFb1L8lM4n4Rz610UzI1E51ZMRG6tuVnphrYnay0/edit#gid=0

**Comment by sophie-delsaux** [comment]
Progress: Reviewed the issue and draft and started to give feedback
Blockers: I have overall feedback but the format only allows me to provide detailed feedback. 
Not sure if the "who benefits" column is really for who would benefit from the glossary or based on where the terms appear.
Availability: 2h
ETA: this week
Pictures or link:
https://docs.google.com/spreadsheets/d/10omnFb1L8lM4n4Rz610UzI1E51ZMRG6tuVnphrYnay0/edit#gid=0

**Comment by bradyse** [comment]
@sophie-delsaux 

Thanks for your work on this. The "who benefits" column is who would benefit from knowing the term. For example, the "TWE" acronym would benefit our team but "content analysis" would benefit anyone within the organization of HfLA.

If you have any general comments, you can add them in the first row of the next column over.

**Comment by Nat-nn** [comment]
Progress: Added Term the definition (per suggestions from the team meetings). After All team review it was decided to remove ' column blocking', this way we can see the term right away (and there is no need to add the term itself into the definition)
Going to leave this issue now and work on a different issue. 

Blockers: Waiting for 2nd "Peer Review".

Availability: 1-3 hours per week
ETA: Leaving this issue, going to work on some other issue. 

Pictures or links:
https://docs.google.com/spreadsheets/d/10omnFb1L8lM4n4Rz610UzI1E51ZMRG6tuVnphrYnay0/edit#gid=0

**Comment by bradyse** [comment]
@Nat-nn 

Please go through each description and revise the formatting such that the description starts with the term in bold and is followed by a _verb_ (not a hyphen).

✅ TWE Intern is in the correct format.
❌ Mentor uses the incorrect format.

![image](https://user-images.githubusercontent.com/74688064/203446645-1d6be233-0eb1-4ff4-9741-ea6ca99ee53c.png)

**Comment by Nat-nn** [comment]
Progress: Updated term formatting (per review comments).  
Blockers: Waiting for Team lead approval
Availability: 1-3 hours per week
ETA: after Holidays (January 2023)
Pictures or links:
https://docs.google.com/spreadsheets/d/10omnFb1L8lM4n4Rz610UzI1E51ZMRG6tuVnphrYnay0/edit#gid=0

**Comment by dialectic51** [comment]
I am currently conducting a peer review of the [Glossary Draft spreadsheet.](https://docs.google.com/spreadsheets/d/10omnFb1L8lM4n4Rz610UzI1E51ZMRG6tuVnphrYnay0/edit#gid=0) Is there any reason why the terms aren't in alphabetical order? Should they be?

**Comment by dialectic51** [comment]
@pandanista @bradyse @Nat-nn 

I have completed my peer review of the [draft of the glossary for the wiki page.](https://docs.google.com/spreadsheets/d/10omnFb1L8lM4n4Rz610UzI1E51ZMRG6tuVnphrYnay0/edit#gid=0) All of my feedback is limited to the Peer Review Questions and Feedback column, with the exception of eliminating extra spaces or adding periods to the end of sentences (or, in rare instances, some very light editing). In the Peer Review column I indicate which component of the wiki glossary requires revision. When necessary, I provide links to sources to justify suggested revisions.

I believe that the next step is for the UX Research Lead to sign off on the peer review. According to the [Sara's comment](https://github.com/hackforla/internship/issues/306#issuecomment-1277955339), I'm also supposed to reach out to the original owner of the issue to see if they can complete the issue. However, I don't think that Natalia will have time to do this. If necessary, I could make the revisions, but since I currently have a number of other items on my plate (issue 338 and some guides), I think it would make the most sense to assign it to someone else--the process should be pretty straightforward.

**Comment by dialectic51** [comment]
Provide Update
1. Progress: Completed.
2. Blockers: Requires sign off by UX Research Lead; original owner (or new UXR) will then complete the issue.
3. Availability: As needed.
4. ETA: Completed

**Comment by pandanista** [comment]
UX lead reviewed the [glossary](https://docs.google.com/spreadsheets/d/10omnFb1L8lM4n4Rz610UzI1E51ZMRG6tuVnphrYnay0/edit#gid=0) as of 1/19, and discussed the first 11 terms during the team meeting on 1/20. Waiting on product to review.

**Comment by priyatalwar** [comment]
I was under the weather, but I've reviewed the comments. I need more time to review rest of the doc. I should have it ready for sign-off at our 2/6 meeting.

**Comment by ExperimentsInHonesty** [comment]
@priyatalwar, I have added a new column for proposed description.  Please use that to take the `Peer Reviewer Questions and Feedback` and provide a revised description (if applicable).

**Comment by priyatalwar** [comment]
2/6 Added proposed description where applicable. We will go review on 2/8 meeting.

**Comment by cbutcher318** [comment]
Progress: added a column with new proposed descriptions, ready for review

**Comment by ExperimentsInHonesty** [comment]
@cbutcher318 Please bring this up at our very next meeting (where I attend)

**Comment by pandanista** [comment]
We will add t-test and paired t-test to the glossary per #375.

**Comment by pandanista** [comment]
Assigning @adarosh to this issue so she could add some statistic concepts such as t-test into the glossary.

**Comment by pandanista** [comment]
T-test, paired sample t-test, independent sample t-test, significance test, N0, descriptive, inferential, Chi-square, mean, median, SD

**Comment by pandanista** [comment]
Please provide update:
1. Progress: "What is the current status of your project? What have you completed and what is left to do?" - we know you just came back from break, please just summarize the progress from before then
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures or links* (if necessary): "Add any pictures or links that will help illustrate what you are working on."

* remember to add links to the top of the issue if they are going to be needed again.

**Comment by pandanista** [comment]
@adarosh The review link is for product to reference where the term was used or discussed to provide more context. For paired t-test, we can refer to issue #375 and [RP019 research roadmap](https://docs.google.com/document/d/1BKFiKRcqDovcaKsEqtWMPU8vUs_Mq3gjNYLU6A5E7dQ/edit#heading=h.4qi601o1qcet). Hope it helps.

**Comment by adarosh** [comment]
Progress Update:

Progress: Added the following terms with their definitions to the glossary: t-test, paired samples t-test, independent samples t-test, statistical significance, hypothesis testing, null hypothesis, descriptive statistics, inferential statistics, chi-square test, mean, median, standard deviation.
Blockers: None
Availability: N/A
ETA: N/A
Pictures (if necessary): N/A

**Comment by pandanista** [comment]
@priyatalwar I've reviewed the definitions of a handful statistical concepts we have decided to include for the glossary. 

Please see the newly added rows filled in blue color in column A of the [Glossary Draft spreadsheet](https://docs.google.com/spreadsheets/d/10omnFb1L8lM4n4Rz610UzI1E51ZMRG6tuVnphrYnay0/edit#gid=0). Let me know if you have any questions. 

If Product approves the updates, then we can move on to the next step to include the glossary to the wiki page.

Thank you.

**Comment by sunannie27** [comment]
Next step: copy content to new columns

**Comment by sunannie27** [comment]
1. Finished reorganizing proposed descriptions and feedback in alphabetical order to match the terms. Reviewed the who benefits column and made edits/comments, but need some clarification.
2. Need clarification on an item that I commented on. 
3. I'll have 5-10 hours total this week to work on issues. 
4. Will be done as soon as I get help/clarification on my one comment.

**Comment by sunannie27** [comment]
1. Progress: Have to go through most of the descriptions and/or write my own
2. Blockers: None
3. Availability: 1 hour
4. ETA: Within 2 weeks

**Comment by ExperimentsInHonesty** [comment]
KC will do another review

**Comment by KC-skc** [comment]
Review completed. Ready for final product review.

**Comment by ExperimentsInHonesty** [comment]
@KC-skc Do you think we need the word ultimately in this definition?

> **End User** is the person who ultimately uses the product or service.

**Comment by sunannie27** [comment]
@KC-skc 
Before
> Card Sorting is a UX research method where study participants group "words" or "phrases" written on notecards (physically or virtually) according to categories that make sense to them. Card sorting could be open (participants make up the categories), closed (researchers provide the categories), and hybrid (a mix of the two).

Your proposed description
> Card Sorting is a research method commonly used in UX where participants categorize "words" or "phrases" presented on notecards (physically or virtually) according to categories that make sense to them. Card sorting could be open (participants make up the categories), closed (researchers provide the categories), or hybrid (combining both).

Why did you change UX research method to a research method commonly used in UX?

**Comment by ExperimentsInHonesty** [comment]
@KC-skc For this definition

> **Evaluative Research** is a research method used for assessing whether a product or proposed solutions addresses users' wants and needs.

I found the following other links that say it's not a method
https://www.questionpro.com/blog/evaluation-research-definition-methods-and-examples/
Evaluation research, also known as program evaluation, refers to research purpose instead of a specific method. Evaluation research is the systematic assessment of the worth or merit of time, money, effort and resources spent in order to achieve a goal.


[Google search for Evaluative Research](https://www.google.com/search?q=evaluative+research+definition&sca_esv=391f93f6c7aeb7e3&ei=QZ0FZumpINjDkPIPidC_4A8&oq=evaluative+research+defin&gs_lp=Egxnd3Mtd2l6LXNlcnAiGWV2YWx1YXRpdmUgcmVzZWFyY2ggZGVmaW4qAggAMgsQABiABBiKBRiRAjIGEAAYFhgeMgYQABgWGB4yBhAAGBYYHjIGEAAYFhgeMgYQABgWGB4yBhAAGBYYHjIGEAAYFhgeMgYQABgWGB4yBhAAGBYYHkieGlDDAliZDHABeAGQAQCYAYIBoAH2BKoBAzMuM7gBAcgBAPgBAZgCB6ACkwXCAgoQABhHGNYEGLADwgINEAAYgAQYigUYQxiwA8ICChAAGIAEGIoFGEPCAgUQABiABMICCBAAGBYYHhgPmAMAiAYBkAYKkgcDNC4zoAe2MA&sclient=gws-wiz-serp), where the previews seem to indicate same

**Comment by sunannie27** [comment]
@KC-skc 
> Foundational research (also referred to as generative research, exploratory research, pathfinding research, or discovery research) attempts to identify and solve a real human problem. The problem centers on a topic or area that has not been clearly defined or explored in the past.
Do you agree that generative research is the same as foundational research? If so, I think we can just put "see foundational research" in the definition for generative research instead of its own definition

**Comment by ExperimentsInHonesty** [comment]
@KC-skc 

> **Exit Interview** is a meeting between management representatives and a person who is leaving an organization or project. The purpose of an exit interview is generally to incorporate the findings to improve the program or product.

Do we need to specify (can be researchers)?
**Exit Interview** is a meeting between management representatives (can be researchers), and a person who is leaving an organization or project. The purpose of an exit interview is generally to incorporate the findings to improve the program or product.

**Comment by KC-skc** [comment]
End User

>Do you think we need the word ultimately in this definition?

>End User is the person who ultimately uses the product or service.

@ExperimentsInHonesty I think it could go either way. Since the original word is "End" User, it's okay to emphasize with ultimately, but it can go either way.

**Comment by KC-skc** [comment]
Card Sorting

>Before : Card Sorting is a UX research method where study participants group "words" or "phrases" written on notecards (physically or virtually) according to categories that make sense to them. Card sorting could be open (participants make up the categories), closed (researchers provide the categories), and hybrid (a mix of the two).

>Your proposed description: Card Sorting is a research method commonly used in UX where participants categorize "words" or "phrases" presented on notecards (physically or virtually) according to categories that make sense to them. Card sorting could be open (participants make up the categories), closed (researchers provide the categories), or hybrid (combining both).

>Why did you change UX research method to a research method commonly used in UX?

@ExperimentsInHonesty In psychology, there is a evaluative method often used in research called the Wisconsin Card Sorting Test. When I hear "card sort", I think of that first so I thought there could be uses of card sorting outside of UX. In a quick Google search, it seems most of the sites equate card sorting to UX, so again, I think it's a judgement call. It might be fine to leave it as it was, or broaden the definition.

**Comment by KC-skc** [comment]
Evaluative Research

>Evaluative Research is a research method used for assessing whether a product or proposed solutions addresses users' wants and needs.
>I found the following other links that say it's not a method
>https://www.questionpro.com/blog/evaluation-research-definition-methods-and-examples/
>Evaluation research, also known as program evaluation, refers to research purpose instead of a specific method. Evaluation research is the systematic assessment of the worth or merit of time, money, effort and resources spent in order to achieve a goal.
>[Google search for Evaluative Research](https://www.google.com/search?q=evaluative+research+definition&sca_esv=391f93f6c7aeb7e3&ei=QZ0FZumpINjDkPIPidC_4A8&oq=evaluative+research+defin&gs_lp=Egxnd3Mtd2l6LXNlcnAiGWV2YWx1YXRpdmUgcmVzZWFyY2ggZGVmaW4qAggAMgsQABiABBiKBRiRAjIGEAAYFhgeMgYQABgWGB4yBhAAGBYYHjIGEAAYFhgeMgYQABgWGB4yBhAAGBYYHjIGEAAYFhgeMgYQABgWGB4yBhAAGBYYHkieGlDDAliZDHABeAGQAQCYAYIBoAH2BKoBAzMuM7gBAcgBAPgBAZgCB6ACkwXCAgoQABhHGNYEGLADwgINEAAYgAQYigUYQxiwA8ICChAAGIAEGIoFGEPCAgUQABiABMICCBAAGBYYHhgPmAMAiAYBkAYKkgcDNC4zoAe2MA&sclient=gws-wiz-serp), where the previews seem to indicate same

@ExperimentsInHonesty Good point. I didn't change it from the original definition because it is on Google search the first definition from Maze's website. Question Pro says " Evaluative research.... refers to research purpose instead of a specific method. Evaluation research is the systematic assessment of the worth or merit of time, money, effort and resources spent in order to achieve a goal. Evaluation research is a type of applied research, and so it is intended to have some real-world effect. " I updated my definition in the spreadsheet to this:

Evaluative Research is an applied method used for assessing whether a product, program, or proposed solutions addresses a particular goal.

What do you think?

**Comment by KC-skc** [comment]
>Foundational research (also referred to as generative research, exploratory research, pathfinding research, or discovery research) attempts to identify and solve a real human problem. The problem centers on a topic or area that has not been clearly defined or explored in the past.
>Do you agree that generative research is the same as foundational research? If so, I think we can just put "see foundational research" in the definition for generative research instead of its own definition

@ExperimentsInHonesty Yes, that sounds fine. We can pick which of the 2 words have a better definition and have it point to the other.

**Comment by KC-skc** [comment]
>Exit Interview is a meeting between management representatives and a person who is leaving an organization or project. The purpose of an exit interview is generally to incorporate the findings to improve the program or product.
>Do we need to specify (can be researchers)?
>Exit Interview is a meeting between management representatives (can be researchers), and a person who is leaving an organization or project. The purpose of an exit interview is generally to incorporate the findings to improve the program or product.

@ExperimentsInHonesty Do you mean should we consider adding "can be researchers"? I think exit interview is a fairly broad term that can be used in lots of different contexts, so I don't think it's necessary to qualify that researchers can do it specifically?

**Comment by ExperimentsInHonesty** [comment]
- [ ] Add more info to evaluative research.  Use [Evaluative research: Key methods, types, and examples](https://maze.co/guides/ux-research/evaluative/#:~:text=Evaluative%20research%20is%20a%20research,and%20uncovering%20areas%20for%20improvement.) as the source.
  - [ ] Add summative and formative research terms to the glossary if they are missing

**Comment by ExperimentsInHonesty** [comment]
- [ ] Rewrite the exit interview definition to remove management representative and be more inclusive of the other types of exit interviews.

**Comment by ExperimentsInHonesty** [comment]
Find a link for each of the new terms you added which would be enough information for the person viewing it to find examples, or other helpful details.

**Comment by ExperimentsInHonesty** [comment]
- [ ] Formative Research definition should include that it's about identifying the audience.

**Comment by ExperimentsInHonesty** [comment]
- [ ] A/B testing needs to be added to the glossary

**Comment by ExperimentsInHonesty** [comment]
Bonnie left off on her review at Formative Research. Will continue later.

**Comment by ExperimentsInHonesty** [comment]
Please reword this so that it doesn't have the word through twice in one sentence

>**Surveys** are a method of examination through collecting information from a sample of individuals through their responses to questions. Both quantitative and qualitative strategies can be used.

**Comment by ExperimentsInHonesty** [comment]
- [ ] Separate Git/ GitHub into two references

**Comment by ExperimentsInHonesty** [comment]
- [ ] update GitHub Contributor

> GitHub Contributor is someone who adds or alters elements of a project on Github.

we need to be more explicit
Code contributors
PR reviews
Documentation
Issue creation
Project Management

**Comment by ExperimentsInHonesty** [comment]
Please find the Provence for this statement under GitHub Contributors

>In GitHub, when a user's e-mail is found in the commit history of a GitHub repo, the user is marked as a contributor.

**Comment by ExperimentsInHonesty** [comment]
We need two definitions in one.  We need to describe how GitHub defines contributors and how Hack for LA defines contributors.

The defines Hack for LA version
A GitHub Contributor is someone who adds or alters GitHub projects or elements such as code, pull requests, documentation, issue creation, project management, comments, etc. 

- [ ] find what contributions qualify by GitHub standards as a contributor.
GitHub definition
In GitHub, when a user's e-mail is found in the commit history of a GitHub repo, the user is marked as a contributor.

**Comment by KC-skc** [comment]
Provenance for "Contributor" in [Github Glossary](https://docs.github.com/en/get-started/learning-about-github/github-glossary).

"A contributor is someone who does not have collaborator access to a repository but has contributed to a project and had a pull request they opened merged into the repository."

**Comment by ExperimentsInHonesty** [comment]
[collaborator](https://docs.github.com/en/get-started/learning-about-github/github-glossary#collaborator)
A collaborator is a person with read and write access to a repository who has been invited to contribute by the repository owner.

**Comment by ExperimentsInHonesty** [comment]
### Internship definitions

Federal:  https://www.dol.gov/agencies/whd/fact-sheets/71-flsa-internships
State: Still looking, but this website says that internships need to be registered with the[ California Department of Industrial Relations](https://www.dir.ca.gov/) https://www.job-law.com/are-unpaid-internships-legal-in-california/



---
### Internship Grant program
https://www.dir.ca.gov/das/Youth-Apprenticeship.html

**Comment by ExperimentsInHonesty** [comment]
we left off on kanban

**Comment by ExperimentsInHonesty** [comment]
we reviewed Kanban Board and @KC-skc will continue to revise.

**Comment by KC-skc** [comment]
Term "Kanban Board" revised.

**Comment by KC-skc** [comment]
BW review left off at "Mockup"

**Comment by ExperimentsInHonesty** [comment]
Let's discuss this 

>Onboarding is the process of integrating a new employee into an organization or familiarizing a new customer or client with one's products or services.

**Comment by ExperimentsInHonesty** [comment]
Please add an example or otherwise make this clearer

Paired t-test
Paired t-test (or Paired sample t-test) is a statistical method which compares the means of two related groups (e.g., same participants tested on the dependent variable twice at different time points).

Notes: we don't define what `dependent variable` means?

**Comment by ExperimentsInHonesty** [comment]
archetype and personas glossary entries

Current Glossary entry:
Persona is a user archetype, often with description that helps designers and developers empathize with their users' business and personal contexts.

Please review this article https://www.nngroup.com/articles/personas-archetypes/ and revisit this definition and make sure we have both

**Comment by ExperimentsInHonesty** [comment]
Please rewrite

User Experience encompasses all aspects of the end-user's interaction with the company, its services, its products, and what the user feels and thinks about the product. User experience is about what users both think and feel. User experience also depends on the context in which the product is used.

**Comment by ExperimentsInHonesty** [comment]
Please revise to just be KPI instead of UX KPI

UX KPIs (or Key Performance Indicator) are the end results, data, and metrics that describe how well the UX solutions succeeded in fulfilling expectations of the designer, team or organization.

**Comment by ExperimentsInHonesty** [comment]
> User Experience (UX) Research is the study of learning what end users of a system or product need and want, for employing those insights to enhance the design process for products, services, or software. UX research can take different forms depending on the area of focus.

>User Experience (UX) Research is the study of learning what end users of a system or product need, desire and expect, then employing those insights to enhance the design process for products, services, or software. UX research can take different forms - such as user interviews and surveys - depending on the question and area of focus.

**Comment by ExperimentsInHonesty** [comment]
- [ ] rewrite this so that it includes when its just a portion of the project 

> Prototype is a model of the proposed final product used for testing before launch. Low-level fidelity (or low-fidelity) prototypes showcase a basic sketch of how a design will look while high-level fidelity (or high-fidelity) prototypes add more details without being a full-design mockup.

**Comment by ExperimentsInHonesty** [comment]
I left off on Qualitative Analysis

**Comment by KC-skc** [comment]
@ExperimentsInHonesty Above comments addressed. Please continue to review.

**Comment by ExperimentsInHonesty** [comment]
What is missing from this definition of Qualitative Analysis is that it's the analysis of free form word answers shared by the research participant
>Qualitative Analysis is a research methodology used to anwer questions and often used to understand the motivations, thoughts, and attitudes of a target audience. Qualitative analysis can involve coding, content analysis, and thematic analysis of oral or written data to reach conclusions.

**Comment by KC-skc** [comment]
@bonniewolfe 

Edited Persona to add archetype. Please let me know if this is acceptable, or whether you'd prefer a separate entry for 'archetype'.

>Persona is a detailed fictitious profile. These descriptions helps managers, designers and developers empathize with their product users' business and personal goals, needs and behaviors. While personas are specific, in comparison, archetypes label abstract characteristics.

**Comment by ExperimentsInHonesty** [comment]
- I am working on this issue https://github.com/hackforla/knowledgebase-content/issues/54 which is related.  I should have the Glossary finished by Thursday end of day 2025-02-13

**Comment by ExperimentsInHonesty** [comment]
Please revise the definition for: Research Plan
to include sharing it with participants (and guardians/parents) after the plan is finalized and before they agree to participate.

**Comment by ExperimentsInHonesty** [comment]
- [ ] Please rewrite Research Repository

**Comment by ExperimentsInHonesty** [comment]
Person mentions archetypes but archetypes is not in the glossary, please add it.

**Comment by ExperimentsInHonesty** [comment]
>Statistical significance is a measure of the probability of the null hypothesis being true compared to the acceptable level of uncertainty regarding the true answer.

If null hypothesis can be described in plain language please do that, otherwise you will have to add null hypothesis as a term

**Comment by ExperimentsInHonesty** [comment]
When you are done updating those, please add this issue to the agenda, so I can sign off on it.

**Comment by ExperimentsInHonesty** [comment]
@KC-skc it looks like this comment is the only one outstanding

>Persona mentions archetypes but archetypes is not in the glossary, please add it.

**Comment by ExperimentsInHonesty** [comment]
@KC-skc I think we just need to change this a little

>Persona is a detailed fictitious profile. These descriptions helps managers, designers and developers empathize with their product users' business and personal goals, needs and behaviors. While personas are specific, in comparison, archetypes label abstract characteristics.

Maybe add an e..g to the end where we name an abstract characteristic

**Comment by ExperimentsInHonesty** [comment]
When this issue is closed, reassign @KC-skc and @ExperimentsInHonesty for credit
